crates/oxc_linter/src/snapshots/no_render_return_value.snap

```diff
@@ -0,0 +1,58 @@
+---
+source: crates/oxc_linter/src/tester.rs
+expression: no_render_return_value
+---
+  ⚠ eslint-plugin-react(no-render-return-value): Do not depend on the return value from ReactDOM.render.
+   ╭─[no_render_return_value.tsx:1:1]
+ 1 │ var Hello = ReactDOM.render(<div />, document.body);
+   ·             ───────────────────────────────────────
```

Comment:
It would be great to point to only `ReactDOM.render` and possibly the assignment operator, since these two are the focus of the error message.